tests/conftest.py

```diff
@@ -397,3 +470,19 @@ def mock_repo_by_name(owner, name):
                 gh.init_account()
             db.session.expire(remote_token.remote_account)
             yield mock_api
+
+
+@pytest.yield_fixture
+def cli_run(app):
+    """Fixture for CLI runner function.
+
+    Returns a function accepting a single parameter (CLI command as string).
+    """
+    runner = CliRunner()
+    script_info = ScriptInfo(create_app=lambda info: app)
+
+    def run(command):
```

Comment:
Normally, we use everywhere the `invoke` method directly. IMHO it's better to stay with that syntax so it's easy for people to switch around.